jobbergate-api/jobbergate_api/apps/job_scripts/routers.py

```diff
@@ -216,6 +217,15 @@ async def job_script_delete(job_script_id: int = Query(..., description="id of t
     """
     Delete job_script given its id.
     """
+    logger.debug(f"Unlinking job_submissions with links to job_script {job_script_id=}")
```

Comment:
same comment I made previously applies here

Reply:
Resolved in commit a308098